src/components/graph/graph-types.ts

```diff
@@ -0,0 +1,29 @@
+export type uuid = string
+export type slug = string
+
+export type nodeData = {
+  slug: string
+  uuid: uuid
+  readonly concepts: Array<string>
+  readonly prerequisites: Array<string>
+}
+
+export type edgeData = {
```

Comment:
```suggestion
export interface EdgeData {
```

Reply:
turns out I am not using this anymore. it's been deleted